modules/location/LocationTypes.ts

```diff
@@ -0,0 +1,9 @@
+export interface Coordinate {
+    lat: number,
+    long: number
+  }
+  
+export interface Location {
```

Comment:
add nalang ang `distance` prop di ?

Reply:
if so need ta gid mag add distance for every object sa listing

Reply:
ara man and distance prop sa line 26 ?

Reply:
additional lang thru map. mas gus2 mo set nalang property?

Reply:
hmmmm, majo budlay gidman mag butang distance sa location hehe

what if make ta wrapper for `location` called `interface locationWithDistance` as return type ka sorter and `findLocationsNearby`?